src/extensions/mvs/components/custom-label/representation.ts

```diff
@@ -38,12 +39,12 @@ export function CustomLabelRepresentation(ctx: RepresentationContext, getParams:
 /** A thingy that is needed to register representation type "Custom Label", allowing user-defined labels at at user-defined positions */
 export const CustomLabelRepresentationProvider = StructureRepresentationProvider({
     name: 'mvs-custom-label',
-    label: 'Custom Label',
+    label: 'MVS Custom Label',
     description: 'Displays labels with custom text',
     factory: CustomLabelRepresentation,
     getParams: () => CustomLabelParams,
     defaultValues: PD.getDefaultValues(CustomLabelParams),
     defaultColorTheme: { name: 'uniform' },
     defaultSizeTheme: { name: 'physical' },
-    isApplicable: (structure: Structure) => structure.elementCount > 0
+    isApplicable: (structure: Structure) => structure.elementCount > 0 && !!structure.models.some(m => m.customProperties.has(MVSAnnotationsProvider.descriptor)),
```

Comment:
I would revert this change, CustomLabelRepresentationProvider should be applicable even without MVSAnnotationsProvider.

(That's also the reason I didn't use the 'MVS ' prefix in the label, but I don't have a strong opinion about it)